src/pages/AddLiquidity/index.tsx

```diff
@@ -873,9 +881,17 @@ function AddLiquidity() {
               <div>
                 <DynamicSection disabled={invalidPool || invalidRange || (noLiquidity && !startPriceTypedValue)}>
                   <AutoColumn gap="md">
-                    <ThemedText.DeprecatedLabel>
-                      {hasExistingPosition ? <Trans>Add more liquidity</Trans> : <Trans>Deposit Amounts</Trans>}
-                    </ThemedText.DeprecatedLabel>
+                    <Row justify="space-between">
+                      <ThemedText.DeprecatedLabel>
+                        {hasExistingPosition ? <Trans>Add more liquidity</Trans> : <Trans>Deposit Amounts</Trans>}
+                      </ThemedText.DeprecatedLabel>
+
+                      <UseWrappedCurrencyToggle
```

Comment:
should we show this toggle if neither token is native or wrapped native?

Reply:
i don't think so. i've the user can't take action on it, it should be hidden